ConfigSpace/configuration_space.pyx

```diff
@@ -1507,11 +1583,11 @@ class ConfigurationSpace(collections.abc.Mapping):
                     substituted_forbidden = forbidden_type(hyperparameter=new_hyperparameter, values=forbidden_arg)
                 else:
                     raise AttributeError(f'Did not find the expected attribute in forbidden {type(forbidden)}.')
-                
+
                 new_forbiddens.append(substituted_forbidden)
             else:
                 raise TypeError(f'Did not expect the supplied forbidden type {type(forbidden)}.')
-        
+
```

Comment:
Great to see all these formatting improvements - did you manage to set up the tooling for Cython files?

Reply:
No I didn't do anything on that front, I just ran a command I have to `StripTrailingWhitespace` that I usually use on files that have it